packages/frontend/src/components/Banner/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { useEffect, useState } from 'react';
+import { useBannerStore } from '../../store/bannerStore';
+import { Alert, Collapse, IconButton } from '@mui/material';
+import { Colors } from '../../constants/Colors';
+import OpenInNewIcon from '@mui/icons-material/OpenInNew';
+import InfoIcon from '@mui/icons-material/Info';
+import './styles.css';
+
+export const Banner = () => {
+  const banners = useBannerStore((state) => state.banners);
+  const getBanners = useBannerStore((state) => state.getBanners);
+
+  const [open, setOpen] = useState(true);
+
+  useEffect(() => {
+    getBanners();
+  }, [getBanners]);
+
+  return (
+    <>
+      <div style={{ margin: '20px 32px 0 32px' }}>
+        <IconButton onClick={() => setOpen(!open)} className="banner-alert">
+          <InfoIcon color="primary" />
+        </IconButton>
+      </div>
+      {banners?.map((banner) => (
```

Comment:
What if users create multiple notices and set them to be shown as banners? Should we allow only a single banner or just a few banners to be active at same time?